app/main.py

```diff
@@ -16,4 +16,12 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+list_of_mutable = [
+    my_favourite_films, marks, collection_of_coins
+]
+list_of_immutable = [
+    lucky_number, pi, one_is_a_prime_number, name, profile_info
+]
+sorted_variables = {
+    "mutable": list_of_mutable, "immutable": list_of_immutable
+}
```

Comment:
it can be, but doesn't such code seem more complex to understand? Simple is best :)

Reply:
should i change code logic or is it about names of variables?